### ISSUE TITLE
Song mode - loop using multi consecutive slots

### ISSUE THREAD

**Issue by ketumatek** [polyend/TrackerIssues#1306]
hi; 
 it will be very nice to put this new feacture in song mode.

- have the possibility to loop blocks as 2 , 3, or more slots at the same time . to improve creativity i need to loop few slot to play on it .
for the moment i can loop only one slot, and it is very unuselfull always had to stop the sequencer go back play the few loops and doing that again. 

- as the same way i select the slots to make a copy or delete them i can select the slot i need and push shift+play to play them in a loop mode.
- please consider this request as something very important and really essentiel for live creativity.

**Comment by tfspsound** [comment]
Agreed. Pressing Play twice quickly to make the the song position jump back is needlessly wearing out the Play button too.

**Comment by colonel-blimp** [comment]
Good idea.  Similar issues: #598, #763
